.github/workflows/build.yml

```diff
@@ -31,11 +65,28 @@ jobs:
         VERSION=local make container
         kind load docker-image ${QUAY_PATH}:local
         make test-e2e
-    - name: Login and push to Quay
-      if: github.event_name == 'push'
-      run: |
-        docker login quay.io -u="${{ secrets.QUAY_USERNAME }}" -p="${{ secrets.QUAY_PASSWORD }}"
-        TAG="$(git rev-parse --abbrev-ref HEAD | tr / -)-$(date +%Y-%m-%d)-$(git rev-parse --short HEAD)"
-        ID="$(docker images ${QUAY_PATH}:local --format "{{.ID}}")"
-        docker tag $ID ${QUAY_PATH}:$TAG
-        docker push ${QUAY_PATH}:$TAG
+  publish:
+    runs-on: ubuntu-latest
+    name: Publish container image to Quay
+    if: github.event_name == 'push'
+    needs:
+      - check-license
+      - generate
+      - build
+      - unit-tests
+      - e2e-tests
```

Comment:
since there are some e2e flakes, should I remove this requirement so that the image publication doesn't fail because of the flakes?

Reply:
I think it is reasonable to comment this out with an explanation of why it is commented out.

Reply:
Done. Thank you for the suggestion.